fdm-calculator/src/balance/nitrogen/emission/ammonia/fertilizers.ts

```diff
@@ -76,6 +76,9 @@ export function calculateNitrogenEmissionViaAmmoniaByFertilizers(
                     acc.mineral.total = acc.mineral.total.add(applicationValue)
                     acc.mineral.applications.push({
                         id: application.p_app_id,
+                        p_id_catalogue: application.p_id_catalogue,
```

Comment:
To be consistent in the output of the balance calculation, only include the id of the application and value. The other properties can be retrieved in the chart from the data object that is already there